rita/src/rita_client/dashboard/wifi.rs

```diff
@@ -144,3 +153,97 @@ impl Handler<GetWifiConfig> for Dashboard {
         Ok(interfaces)
     }
 }
+
+pub fn set_wifi_ssid(wifi_ssid: Json<WifiSSID>) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    debug!("/wifi_settings/ssid hit with {:?}", wifi_ssid);
+
+    let wifi_ssid = wifi_ssid.into_inner();
+    let mut ret: HashMap<String, String> = HashMap::new();
+
+    if let Err(e) = validate_config_value(&wifi_ssid.ssid) {
+        info!("Setting of invalid SSID was requested: {}", e);
+        ret.insert("error".to_owned(), format!("{}", e));
+        return Box::new(future::ok(
+            HttpResponse::new(StatusCode::BAD_REQUEST)
+                .into_builder()
+                .json(ret),
+        ));
+    }
+
+    Box::new(
+        Dashboard::from_registry()
+            .send(wifi_ssid)
+            .from_err()
+            .and_then(move |_reply| future::ok(HttpResponse::Ok().json(ret))),
+    )
+}
+
+pub fn set_wifi_pass(wifi_pass: Json<WifiPass>) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    debug!("/wifi_settings/pass hit with {:?}", wifi_pass);
+
+    let wifi_pass = wifi_pass.into_inner();
+    let mut ret: HashMap<String, String> = HashMap::new();
```

Comment:
Rust should be able to infer types so it would be safe to do `let mut ret = HashMap::new()`